### PR TITLE
[Issue-789] Fixed Native Search doesn't return complete object issue - 2.0

### PR DESCRIPTION
**Pull request by shiva-beehyv**

### CODE REVIEW

**Review by Sreejit-K — APPROVED** [review]

**Comment by holashchand** [comment]
Multiple search operators not working
i.e
```
{
  "offset": 0,
  "limit": 2,
  "filters": {
    "policyDetails": {
        "policyNumber": {
            "contains": "insurance10"
        }
    },
    "mobile": {
        "contains": "insurance101"
    }
  }
}
```

**Review by holashchand — APPROVED** [review]

**Comment by holashchand** [comment]
creating a new ticket for the issue i mentioned above

**Comment by holashchand** [comment]
@shiva-beehyv we should create the PR for main branch

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -23,11 +23,14 @@ jobs:
     - name: Set up properties
       run: sh configure-dependencies.sh
 #   # debug step
-    # - name: Setup upterm session
-    #   uses: lhotari/action-upterm@v1
-    #   with:
-    #     ## limits ssh access and adds the ssh public key for the user which triggered the workflow ie Sreejith-K
-    #     limit-access-to-actor: true
+    - name: Setup upterm session
```
Comment (holashchand) [inline .github/workflows/maven.yml:26]: comment this change or revert

**Review by holashchand — CHANGES_REQUESTED** [review]

**Comment by shiva-beehyv** [comment]
Closing this ticket since the initial branching off was done from a wrong branch.
Creating [another pr](https://github.com/Sunbird-RC/sunbird-rc-core/pull/303/commits)

**Pull request closed without merging**